macros/sql/safe_subtract.sql

```diff
@@ -0,0 +1,28 @@
+{%- macro safe_subtract(field_list) -%}
+    {{ return(adapter.dispatch('safe_subtract', 'dbt_utils')(field_list)) }}
+{% endmacro %}
+
+{%- macro default__safe_subtract(field_list) -%}
+
+{%- if field_list is not iterable or field_list is string or field_list is mapping -%}
+
+{%- set error_message = '
+Warning: the `safe_subtract` macro now takes a single list argument instead of \
+string arguments. The {}.{} model triggered this warning. \
+'.format(model.package_name, model.name) -%}
+
+{%- do exceptions.warn(error_message) -%}
```

Comment:
In general, I don't think this note is necessary - we had it for `safe_add` as part of its behaviour migration, but this is a net-new macro that is being built the right way. 

If someone does `{{ dbt_utils.safe_subtract('first', 'second') }}` it will render as `coalesce(f, 0) - coalesce(i, 0), ...` which will send them back to double check the docs anyway. 

So two options: 
- Remove altogether
- Change it to an exception and don't say "now takes", because it never didn't take that form

Reply:
That makes total sense to me. I think since the shift is still fairly new it makes sense to throw an exception and have updated it accordingly, but let me know if you would prefer to remove altogether instead.